SocialFilter/sections/specific.txt

```diff
@@ -61,6 +61,7 @@ vgorode.ua##.social_plugins
 fakty.com.ua##.like_box > li:not(.f_right)
 afmen.online##.wrap-socials_webnavoz
 ipress.ua##.yashare
+ellegirl.ru##.social-container
```

Comment:
just links to blogs. Usually we don't block them

Reply:
They are hiding the widgets which show e. g. their posts, not a link to their e. g. facebook site.

Reply:
I see. @zloyden please also block `http://platform.twitter.com/widgets.js` (add domain to existing rule)